doc/installation/centos.rst

```diff
@@ -4,10 +4,180 @@
+    $ mysql_secure_installation
+
+Setup the database for the privacyIDEA server::
+
+    $ echo 'create database pi;' | mysql -u root -p
+    $ echo 'create user "pi"@"localhost" identified by "<dbsecret>";' | mysql -u root -p
+    $ echo 'grant all privileges on pi.* to "pi"@"localhost";' | mysql -u root -p
+
+If this should be a pinned installation (i.e. with all the package pinned to
+the versions with which we are developing/testing), some more packages need to
+be installed for building these packages::
+
+    $ yum install gcc postgresql-devel
+
+Create the necessary directories::
+
+    $ mkdir /etc/privacyidea
+    $ mkdir /opt/privacyidea
+    $ mkdir /var/log/privacyidea
+
+Add a dedicated user for the privacyIDEA server and change some ownerships::
+
+    $ useradd -r -M -d /opt/privacyidea privacyidea
+    $ chown privacyidea:privacyidea /opt/privacyidea /etc/privacyidea /var/log/privacyidea
+
+Install the privacyIDEA server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now switch to that user and install the virtual environment for the privacyIDEA
+server::
+
+    $ su - privacyidea
+
+Create the virtual environment::
+
+    $ virtualenv /opt/privacyidea
+
+and activate it::
+
+    $ . /opt/privacyidea/bin/activate
+
+If this should be a pinned installation (that is the environment we use to build and test),
+we need to install some pinned dependencies first. They should match the version of the targeted
+privacyIDEA::
+
+        (privacyidea)$ pip install -r https://raw.githubusercontent.com/privacyidea/privacyidea/v3.0.2/requirements.txt
+
+Then just install the targeted privacyIDEA version with::
+
+        (privacyidea)$ pip install privacyidea==3.0.2
+
+Setting up privacyIDEA
+^^^^^^^^^^^^^^^^^^^^^^
+
+In order to setup privacyIDEA a configuration file must be added in
+``/etc/privacyidea/pi.cfg``. It should look something like this::
+
+    import logging
+    # The realm, where users are allowed to login as administrators
+    SUPERUSER_REALM = ['super']
+    # Your database
+    SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://pi:<dbsecret>@localhost/pi'
+    # This is used to encrypt the auth_token
+    #SECRET_KEY = 't0p s3cr3t'
+    # This is used to encrypt the admin passwords
+    #PI_PEPPER = "Never know..."
+    # This is used to encrypt the token data and token passwords
+    PI_ENCFILE = '/etc/privacyidea/enckey'
+    # This is used to sign the audit log
+    PI_AUDIT_KEY_PRIVATE = '/etc/privacyidea/private.pem'
+    PI_AUDIT_KEY_PUBLIC = '/etc/privacyidea/public.pem'
+    PI_AUDIT_SQL_TRUNCATE = True
+    # The Class for managing the SQL connection pool
+    PI_ENGINE_REGISTRY_CLASS = "shared"
+    PI_AUDIT_POOL_SIZE = 20
+    PI_LOGFILE = '/var/log/privacyidea/privacyidea.log'
+    PI_LOGLEVEL = logging.INFO
+
+Make sure the configuration file is not world readable:
+
+.. code-block:: bash
+
+    (privacyidea)$ chmod 640 /etc/privacyidea/pi.cfg
+
+More information on the configuration parameters can be found in :ref:`cfgfile`.
```

Comment:
This ":ref:" just jumps to the start of the page.

Reply:
Same here, it takes me to the inifile page.